test/features/dataset_deletion.feature

```diff
@@ -0,0 +1,36 @@
+@dataset_deletion
+Feature: Dataset deletion
+
+    Scenario: Sysadmin creates a dataset
+        Given "SysAdmin" as the persona
+        When I log in
+        And I go to "/dataset/new"
+        Then I fill in "title" with "Dataset deletion"
+        Then I fill in "notes" with "notes"
+        Then I execute the script "document.getElementById('field-organizations').value=jQuery('#field-organizations option').filter(function () { return $(this).html() == 'Test Organisation'; }).attr('value')"
+        Then I select "False" from "private"
+        Then I fill in "version" with "1"
+        Then I fill in "author_email" with "test@test.com"
+        Then I press "save"
+        And I wait for 10 seconds
+        Then I execute the script "document.getElementById('field-image-url').value='https://example.com'"
+        Then I fill in "name" with "res1"
+        Then I fill in "description" with "description"
+        Then I press the element with xpath "//button[@value='go-metadata']"
+        And I wait for 10 seconds
+        Then I should see "Data and Resources"
+
+    Scenario: Sysadmin deletes a dataset
```

Comment:
Scenario's may not run in order, can you add a different dataset to be created (or check created) and then do the 'delete' step

Reply:
I've not seen this fail before from running out of order, even though it's run in multiple repositories...but I'll take a look.

Should we just combine it with the previous scenario? If creating a dataset is an inherent part of testing deletion, then we may as well, it seems.

Reply:
Id still leave in to explain where it failed. i.e duplicate most of the above test but on a different name/ref

Reply:
Behave still gives us a line-by-line identification of where it failed.